media-sound/qmmp/qmmp-2.2.3.ebuild

```
@@ -0,0 +1,170 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg
+
+DESCRIPTION="Qt-based audio player with winamp/xmms skins support"
+HOMEPAGE="https://qmmp.ylsoftware.com"
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="
+		https://qmmp.ylsoftware.com/files/qmmp/$(ver_cut 1-2)/${P}.tar.bz2
+		https://downloads.sourceforge.net/project/qmmp-dev/qmmp/$(ver_cut 1-2)/${P}.tar.bz2
+
+	"
+	KEYWORDS="~amd64 ~x86"
+else
+	inherit subversion
+	local BRANCH_VER="2.2" # NOTE: bump this to $(ver_cut 1-2) for live ebuild
```

Ah, I didn't see these changes, why a new variable ?
Also you can't use local out of a function.
```suggestion
	BRANCH_VER="2.2" # NOTE: bump this to $(ver_cut 1-2) for live ebuild
```

---

i think you could/should revert here. ebuild don't need more attention for useless changes.

---

the entire block if/fi don't require changes.

---

Empty line or comment are not necessary...
please